db/schema.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+# rubocop:disable Metrics/BlockLength
+ActiveRecord::Schema[7.0].define(version: 20_220_913_154_936) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension 'plpgsql'
+
+  create_table 'comments', force: :cascade do |t|
+    t.text 'text'
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.bigint 'author_id', null: false
+    t.bigint 'posts_id', null: false
+    t.index ['author_id'], name: 'index_comments_on_author_id'
+    t.index ['posts_id'], name: 'index_comments_on_posts_id'
+  end
+
+  create_table 'likes', force: :cascade do |t|
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.bigint 'posts_id', null: false
+    t.bigint 'author_id', null: false
+    t.index ['author_id'], name: 'index_likes_on_author_id'
+    t.index ['posts_id'], name: 'index_likes_on_posts_id'
+  end
+
+  create_table 'posts', force: :cascade do |t|
+    t.string 'title'
+    t.text 'text'
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.bigint 'author_id', null: false
+    t.index ['author_id'], name: 'index_posts_on_author_id'
+  end
+
+  create_table 'users', force: :cascade do |t|
+    t.string 'name'
+    t.string 'photo'
+    t.text 'bio'
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+  end
```

Comment:
- [ ] Please make sure to add column `PostsCounter` to `users` table.

![image](https://user-images.githubusercontent.com/83803414/190001156-b403c7cc-4e42-4021-b9d7-ba3c914cab76.png)